wgsl/index.bs

```diff
@@ -12214,18 +12211,21 @@ WGSL provides two kinds of value constructors:
 
 Each [=type/concrete=], [=constructible=] *T* has a unique <dfn noexport>zero value</dfn>,
 and a corresponding built-in function written in WGSL as the type followed by an empty pair of parentheses: *T* `()`.
+[=Abstract numeric types=] also have zero values, but there are no built-in function to access them.
 
 The zero values are as follows:
 
 * `bool()` is `false`
-* `i32()` is 0
-* `u32()` is 0
-* `f32()` is 0.0
-* `f16()` is 0.0
+* `i32()` is 0i
+* `u32()` is 0u
+* `f32()` is 0.0f
+* `f16()` is 0.0h
 * The zero value for an *N*-component vector of type *T* is the *N*-component vector of the zero value for *T*.
 * The zero value for an *C*-column *R*-row matrix of type *T* is the matrix of those dimensions filled with the zero value for *T*.
 * The zero value for a [=constructible=] *N*-element array with element type *E* is an array of *N* elements of the zero value for *E*.
 * The zero value for a [=constructible=] structure type *S* is the structure value *S* with zero-valued members.
+* The zero value of an [=AbstractInt=] is 0.
```

Comment:
Backticks on the `0` would help separate the full-stop from the literal (here and below)

Reply:
Will be fixed with https://github.com/gpuweb/gpuweb/pull/4485